compiler/rustc_resolve/src/macros.rs

```diff
@@ -960,4 +935,46 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
         let ItemKind::MacroDef(def) = &item.kind else { unreachable!() };
         MacroData { ext: Lrc::new(ext), rule_spans, macro_rules: def.macro_rules }
     }
+
+    fn accessible(
+        &mut self,
+        expn_id: LocalExpnId,
+        path: &ast::Path,
+        namespaces: &[Namespace],
```

Comment:
I think it would be fine to add this argument to `cfg_accessible` instead of introducing a new separate function.

Just need to rename it to something like `path_accessible`.

Reply:
These changes have had some impact: both `﻿rustc_builtin_macros` and ﻿`rustc_expand` had to incorporate `﻿rustc_hir`, where ﻿`enum Namespace` is located.

Reply:
Ah, *that*'s why there were two methods, I see.

I'd rather move `enum Namespace` to some common dependency of `rustc_expand` and `rustc_hir`, but I don't see a good place for it.
Let's keep two methods then, dependency on `rustc_hir` will slow down pipelined builds.